Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -14082,6 +14141,23 @@ end
 					},
 					Notes = "Converts the {{Globals#eDamageType|DamageType}} to a string representation ",
 				},
+				DimensionToString =
+				{
+					Params =
+					{
+						{
+							Name = "Dimension",
+							Type = "Globals#eDimension",
```

Comment:
Does this pass through the PluginChecker's API parsing? I'd expect a simple `eDimension`.